meeshkan/build/paths.py

```diff
@@ -128,9 +128,22 @@ def combine_paths_into_single_path(path0: str, path1: str) -> str:
         return (new_path, path, matches)
     return (request_path, None, None)
 
+# content of mymodule.py
+def something():
+    """ a doctest in a docstring
+    >>> something()
+    42
+    """
+    return 41
```

Comment:
This should fail if discovered.

Reply:
It's copied directly from pytest doctest [documentation](http://doc.pytest.org/en/latest/doctest.html) for testing purposes.